test/cpu_fusion.cpp

```diff
@@ -3069,6 +3069,7 @@ TEST(cpu_fusion, fuse_batch_dot_forward)
     auto cpu_f = make_function_from_file(file_name);
     auto int_f = make_function_from_file(file_name);
     pass_manager.run_passes(cpu_f);
+    pass_manager.run_passes(int_f);
```

Comment:
Why do we need to run fusion on interpreter side?

Reply:
I still need to write more unit tests to over different use cases. But in this case, the input graph hare multiple `Dot` operators that need to be fused into `BatchDot`.